ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -175,8 +179,19 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
         return localizeObs.retry(3);
       })
       .subscribe((nbName) => {
-        this.progress = Progress.Redirecting;
-        this.window.location.href = this.notebookUrl(this.cluster, nbName);
+        this.incrementProgress(Progress.Redirecting);
+        if (this.creating) {
+          window.history.replaceState({}, '', 'workspaces/' + this.wsNamespace +
```

Comment:
Does this clear the window title? (that should be the second parameter)

Reply:
It has not in my testing but I will try doing that.

Reply:
To be clear, I don't care very much whether it updates the window title from "creating" - I just wanted to verify it wasn't setting it to the empty string.